### PR TITLE
master-qa-196973

### PR DESCRIPTION
**Pull request by Songyuewen**
https://liferay.atlassian.net/browse/LPS-196973

### CODE REVIEW

**Comment by Songyuewen** [comment]
Now I add the macro for hide applications can only add one application, would it be better if use add multiple applications?

**Comment by Songyuewen** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/Publications.macro [hunk 1]
```diff
@@ -273,6 +273,14 @@ definition {
 		return ${publicationName};
 	}
 
+	macro hideApplications {
```
Comment (suilin) [inline portal-web/test/functional/com/liferay/portalweb/macros/Publications.macro]: We don't need to add the macro and path, we can use the existing macro `SystemSettings.editTextAreaSetting` and resend it?

**Comment by Songyuewen** [comment]
ci:test:sf

**Comment by suilin** [comment]
ci:forward

**Pull request closed without merging**